Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -1772,7 +1774,19 @@ public void onSuccess(Location location) {
                                         userPositionMarker.setVisible(true);
                                     userPositionMarker.setPosition(latLng);
                                 }
-                                googleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(latLng, zoom));
+                                try {
+                                    if (Objects.equals(zoomType, "NO_ZOOM")) {
```

Comment:
Create `private static final String` constants for string type "ZOOM" , "NO_ZOOM"

Reply:
not required